__tests__/generator/index.test.ts

```diff
@@ -42,6 +42,9 @@ describe('Generator', () => {
     });
   });
   it('generate synthetics project - NPM', async () => {
+    // In a few development environments, this test may take
+    // a few millisseconds more than the default 15000ms timeout
+    jest.setTimeout(30000);
```

Comment:
Its already this way - https://github.com/elastic/synthetics/blob/de8feeec94558b78268d54b2d14f5e60d80c1039/__tests__/generator/index.test.ts#L89